checks/__init__.py

```diff
@@ -322,6 +322,7 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         self.agentConfig = agentConfig
         self.in_developer_mode = agentConfig.get('developer_mode') and psutil
         self._internal_profiling_stats = None
+        self.default_integration_http_timeout = int(agentConfig.get('default_integration_http_timeout', 9))
```

Comment:
Nitpick but let's cast this to a float if someone wants to use a non integer timeout